### DISCUSSION TITLE
Possibility to embed a giu in a winapi window?

### DISCUSSION THREAD

**Discussion by qFlensT** [AllenDang/giu#428] [Q&A] [answered]
Is there any possibility to "embed" this framework into a winapi window? Let me explain my intentions: I want to create a window using the **CreateWindowInBandEx** function using winapi, so that I can turn it into an overlay, that runs on top of full-screen applications and uses the graphical interface of this framework.

**Comment by AllenDang** [comment, marked as answer]
@qFlensT giu is based on https://github.com/go-gl/glfw  and it's impossible to use another backend unless you create a new platfomer.
But if you want to create a top most, borderless and transparent window, try the read the window creation document of GLFW (https://www.glfw.org/documentation) and try the flags with MasterWindow.

**Comment by qFlensT** [comment]
@AllenDang Thanks for your reply. I'll try to think of something. It's just that the **CreateWindowInBand** function was obtained by reverse engineering "user32.dll", and as far as I know, this is the only way to draw a window on top of a full-screen application (no borderless) without using a direct3d scene hook. Here's an example, if anyone is interested: https://github.com/CovERUshKA/Autoclicker. Thanks again, I'll try to find the flags I need.
